### DISCUSSION TITLE
WLAN Pi Pro availability

### DISCUSSION THREAD

**Discussion by ThatIdiotSid** [WLAN-Pi/feedback#22] [Hardware - WLAN Pi Pro] [answered]
I apologize if this is the wrong place to ask, but I didn't see any other place to send this in on the main WLAN Pi site.

Any ETA on when the general public can get those units?   (I know at least one person at the latest WLPC I couldn't attend got an advance unit  that they raved about on their blog)

I'd be more than happy to pony up the money up front like a preorder to get in a queue, since  I'm frothing at the mouth to get my hands on one  (I love having new WLAN toys to play with).    :)

**Comment by jolla** [comment, marked as answer]
Due to the supply constraints, especially with the Raspberry Pi CM4's it is hard to say when units will be readily available.
